### ISSUE TITLE
Player inputted initiative?

### ISSUE THREAD

**Issue by The-Sovereign-Psiocrat** [gvorbeck/codex-quest#354]
Hey, I love your recent additions with the encounter and treasure generators. They save a lot of time from consulting the book. Do you have any plans to add the functionality to let players roll their own initiative for the DM in the round tracker? I have been having to manually input it, and it's been the biggest speed bump in our game. No rush; I am just curious if it's on the roadmap. Also, I appreciate the updates you have made. Codex Quest has been such a big part of my games since I started using it.

**Comment by gvorbeck** [comment]
Yeah, that is something worth investigating. I have to run some tests, but conceivably this is possible. Let me make this my next project. I'm working on an API that will make integrating with Discord apps a bit easier and then I'll check this out. It would be nice if players' clicking the 'Roll Initiative' button actually produced some effect rather than nothing. No promises, but I'll look into it.

**Comment by The-Sovereign-Psiocrat** [comment]
Also so I don't have to make a new issue, for some reason, new characters don't have their bio & notes function at least for me. Also, I don't think armor negates speed in Codex Quest.

**Comment by gvorbeck** [comment]
Thanks for identifying these.

I have fixed the description fields from not appearing.

The armor bug is perplexing. It works for me with custom armor but not for standard armor. I will have to look more closely at this one tomorrow.

**Comment by gvorbeck** [comment]
Movement is fixed. UNFORTUNATELY, custom armors will need to be remade in order to work with the new system. SORRY.

**Comment by The-Sovereign-Psiocrat** [comment]
I was messing around with the custom armor and it still seems not to adjust movement even when remade. Am I overlooking something?

https://codex.quest/u/TqJfMPZPyVe9SJtlIdMmnAk6dHu2/c/H12M3iEH4mgMdiPON8VG

Heres a character I made wearing custom metal armor and their speed is still 40

**Comment by gvorbeck** [comment]
my bad, should be fixed now! I can see your character now at 20'
